.github/build-elasticsearch-image.sh

```diff
@@ -0,0 +1,7 @@
+docker image build \
+    --tag ${DOCKER_USERNAME}/elasticsearch:${DOCKER_ELASTICSEARCH_IMAGE_TAG} \
+    --no-cache \
+    --compress \
+    -f project-base/docker/elasticsearch/Dockerfile \
+    . &&
```

Comment:
the 'and' is not neccessary here now
```suggestion
    .
```